### PR TITLE
Close #30 start anatomy of generated code

### LINKED ISSUE #30: Add: Structure of the ALWeaver YAML file

### PR DESCRIPTION
**Pull request by plocket**
Closes #30 until further review of content happens. Test:
- [ ] page runs and is visible on the site

### CODE REVIEW

File: docs/anatomy_of_weaver_code.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
+---
+id: yaml_anatomy
+title: The Anatomy of an ALWeaver Generated YAML File
+sidebar_label: Anatomy of the YAML
+slug: /generated_yaml
+---
+
+The code [ALWeaver](https://github.com/suffolkLITLab/docassemble-assemblylinewizard) generates is just a starting point. Is uses the [labels and variables you added to your templates](doc_vars_reference.md) to make an interview that uses the AssemblyLine library.
```
Comment (BryceStevenWilley) [inline docs/anatomy_of_weaver_code.md:8]:
```suggestion
The code [ALWeaver](https://github.com/suffolkLITLab/docassemble-assemblylinewizard) generates is just a starting point. It uses the [labels and variables you added to your templates](doc_vars_reference.md) to make an interview that uses the AssemblyLine library.
```

File: sidebars.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ module.exports = {
       items: [
         'pdfs',
         'document_variables_reference',
+        'yaml_anatomy',
```
Comment (BryceStevenWilley) [inline sidebars.js:13]: Minor: IDK if this belongs in the PDFs and DOCXs heading?

File: src/css/custom.css [hunk 3]
```diff
@@ -16,6 +16,11 @@
   --ifm-color-primary-lightest: rgb(146, 224, 208);
   --ifm-code-font-size: 95%;
   --doc-sidebar-width: 15em !important;
+  --ifm-container-width: 100vw;
+}
+
+.container.docItemWrapper_node_modules-\@docusaurus-theme-classic-lib-next-theme-DocPage- {
```
Comment (BryceStevenWilley) [inline src/css/custom.css:22]: Saying this as someone who knows no CSS: is it worth it to comment what our custom CSS is doing?

**Review by BryceStevenWilley — APPROVED** [review]
LGTM

File: sidebars.js (hunk 2 above)
Comment (plocket) [reply sidebars.js:13]: Good question. Created #55 for this

File: src/css/custom.css (hunk 3 above)
Comment (plocket) [reply src/css/custom.css:22]: Also good point. Created #56

**Pull request merged**